packages/icons/src/cancel.svg

```diff
@@ -1,6 +1,6 @@
 <!--
   Based on Material design icons by Google - https://github.com/google/material-design-icons
-  which are licensed under the Apache License 2.0. Icon 'highlight_off (rounded)'.
+  which are licensed under the Apache License 2.0. Icon 'cancel (outlined)'.
```

Comment:
This is wrong to change. We named "cancel" to the "highlight_off" icon of the material icon.

Reply:
I'll undo it.